.gitignore

```diff
@@ -34,3 +34,5 @@ googlemaps.egg-info
 *.egg
 .vscode/
 .idea/
+index.py
```

Comment:
Does this need to be added to gitignore?

Reply:
I need this for local development.

Note: I will submit a new pull request as single commit for all changes to this review round